bowling.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset='utf-8'>
+<script type="text/javascript" src="app/public/bowling_logic.js"></script>
+<script src="https://code.jquery.com/jquery-2.1.4.min.js"> </script>
+<script src="https://code.jquery.com/jquery-migrate-1.2.1.min.js"> </script>
+<link rel="stylesheet" href="./app/public/stylesheets/stylesheet.css">
+<link href='http://fonts.googleapis.com/css?family=Montserrat:400,700' rel='stylesheet' type='text/css'>
+</head>
+
+
+<body>
+
+  <center>
+    <div class='container'>
+      <div class='heading'>
```

Comment:
If you want it to fadeIn, have it hidden at the start. You can use <style='display:none'>